servomaxx: 0.5   #max degree servo horizontal (x) can turn
servomin: -0.5   # Min degree servo horizontal (x) can turn
step_distancex: 0.01 #x servo rotation steps



